storage/repair.go

```diff
@@ -119,8 +119,9 @@ func (r shardRepairer) Repair(
 	metadata.AddLocalMetadata(origin, localIter)
 
 	// Add peer metadata
-	peerIter, err := session.FetchBlocksMetadataFromPeers(namespace,
-		shard.ID(), start, end, result.NewOptions(), client.FetchBlocksMetadataEndpointV2)
+	level := client.ReadConsistencyLevelMajority
```

Comment:
should this be moved to `storage/repair/Options`?

Reply:
Yeah options seems like the right place, although I wonder if we would even want to allow a non-quorum repair...

Reply:
Sure thing, can add there.